platforms/desktop/src/pages/launch/LaunchPage.css

```diff
@@ -0,0 +1,87 @@
+.launchContentContainer {
```

Comment:
Using percentages for padding as well as using scale over font-size might cause weird behavior, consider using fixed rem values instead

Reply:
Replaced with fixed padding and font sizes